com.unity.hlod/Runtime/HLODUserData.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using Codice.Client.BaseCommands;
```

Comment:
I'm not sure what this namespace is for, but this makes building a player or assetbunldes fail.
When I removed the line, I couldn't see any compile error messages or build failure messages.

Reply:
Thanks!. I don't sure why it was there :) 
Fixied.